package.json

```diff
@@ -13,9 +13,12 @@
     "docz:dev": "docz dev",
     "docz:build": "docz build",
     "build:css": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 index.scss -o css/",
+    "build:icon": "yarn run clean:icon && mkdirp packages/de-code-icons/fonts && mkdirp packages/de-code-icons/css && cp icons/fonts/* packages/de-code-icons/fonts/ && cp icons/css/style.css packages/de-code-icons/css/de-code-icons.css",
     "clean": "rm -rf node_modules dist temp",
+    "clean:icon": "mkdirp tmp && mv packages/de-code-icons/*.json packages/de-code-icons/*.md tmp && rm -rf packages/de-code-icons/ && mkdirp packages/de-code-icons && mv tmp/* packages/de-code-icons/ && rm -rf tmp",
     "bootstrap": "lerna bootstrap",
     "publish": "lerna run build:css && lerna publish",
+    "publish:icon": "lerna run build:icon && lerna publish",
```

Comment:
lerna publish will publish all the packages in a mono repo, it will not publish just an individual project.